content/blog/getting-started-with-pac/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "Getting Started With Pac"
```

Comment:
Should this be `PaC`? Do we want to spell it out like `Policy as Code`? Or `Policy as Code (PaC)`?

Reply:
fixed